src/Attribute.h

```diff
@@ -0,0 +1,41 @@
+#ifndef ATTRIBUTE_H
+#define ATTRIBUTE_H
+
+#include <string>
+#include <vector>
+using namespace std;
+
+class AttributeGenerator;
+
+class Attribute						//Base class with pure virtual methods which will be derived as per different types of attribute generation
+{
+public:
+	string attribute;				//Name of attribute
+	int attribute_probability;			//Attribute generation probability
+	Attribute(string, int);
+	virtual string OutputAttributes() = 0;		//Checks attribute probability and generate is accordingly
```

Comment:
Please put the comment on a different line so that we don't have such long lines.
That's last change I see. Thanks.

Reply:
Done. I have placed those lines above respective classes with detailed description for readability. Is it fine now? 

Is pull request ready to merge?

Reply:
> Done. I have placed those lines above respective classes with detailed description for readability. Is it fine now?

No, please put the comments within the class declaration, just above a declaration.

Reply:
Done. Is it fine to merge now?

Reply:
Yes, it's fine. Thank you.